src/tools/dart2native.md

```diff
@@ -1,7 +1,101 @@
 ---
 title: dart2native
 description: Command-line tool for AOT-compiling Dart source code.
-toc: false
 ---
 
-Coming soon.
+Use the `dart2native` command to AOT (ahead-of-time) compile
+a Dart program to [native x64 machine code](/platforms).
+The `dart2native` command is supported on Windows, macOS, and Linux.
+
+{{site.alert.note}}
+  To execute Dart code without AOT compiling it,
+  use [`dart`, the standalone Dart VM](/tools/dart-vm).
+{{site.alert.end}}
+
+The output of `dart2native` is either
+a standalone executable (the default)
+or an AOT snapshot that you can run with the [`dartaotruntime` command][].
+A **standalone executable** is native machine code that's compiled from
+the specified Dart file and its dependencies,
+plus a small Dart runtime that handles
+type checking and garbage collection.
+
+An **AOT snapshot** doesn't include the Dart runtime.
+Consider using snapshots if you're distributing multiple programs
+and disk space is limited.
+
+
+## Creating standalone executables
+
+Here's an example of using `dart2native` to create a standalone executable:
+
+```terminal
+$ dart2native bin/main.dart -o bin/my_app
+```
+
+You can distribute and run that executable like you would
+any other executable file:
+
+```terminal
+$ cp bin/my_app ~/bin
+$ my_app
```

Comment:
I believe you need `./my_app` (current dir is not in path on anything but windows)

Reply:
Ah, but I copied it to ~/bin, which is in MY path. :)

But you're probably right that this is too subtle.

Reply:
I've changed it to be /usr/local/bin instead of ~/bin. OK, @mit-mit?

Reply:
Most Linux users don’t have write permissions on /usr/local/bin. Please revert.

Reply:
OK... So @mit-mit are you OK with the previous text? It works. (In my setup, at least.)

Reply:
Yes, `/usr/local/bin` is not a good choice; Windows users are going to have no understanding of that that is.

I suggest you simply:

```terminal
cp bin/my_app .
./my_app
```